internal/signaling/peer.go

```diff
@@ -207,35 +207,26 @@ func (p *Peer) HandleHelloPacket(ctx context.Context, packet HelloPacket) error
 		logger.Info("peer connecting", zap.String("game", p.Game), zap.String("peer", p.ID))
 	}
 
-	if hasReconnected && len(reconnectingLobbies) > 0 && reconnectingLobbies[0] != "" {
-		lobby := reconnectingLobbies[0]
-		inLobby, err := p.store.IsPeerInLobby(ctx, p.Game, lobby, p.ID)
-		if err != nil {
-			return err
-		}
-		if inLobby {
+	err := p.Send(ctx, WelcomePacket{
+		Type:   "welcome",
+		ID:     p.ID,
+		Secret: p.Secret,
+	})
+	if err != nil {
+		return err
+	}
+
+	if hasReconnected {
+		for _, lobbyID := range reconnectingLobbies {
 			logger.Info("peer rejoining lobby", zap.String("game", p.Game), zap.String("peer", p.ID), zap.String("lobby", p.Lobby))
-			p.Lobby = lobby
+			p.Lobby = lobbyID
```

Comment:
We still have only one `Lobby` on peer, so we still don't really support being in multiple lobbies.